### ISSUE TITLE
Weird right handshape misalignment Rachel 65, U7 "FINISH"

### ISSUE THREAD

**Issue by geograil** [DCS-LCSR/SignStream3#115]
File: Rachel 65, U7

The right handshape isn't aligned with the gloss. I tried to unlock the gloss-phon alignment and set just that handshape start time to be the same and then would re-lock it, but it wouldn't move it that way either and gave me an error message. I also tried to do what it said and unlocked the hand alignments, but it wouldn't let me change it that way either. It gave me the same error message. 

![screen shot 2017-01-27 at 3 39 53 pm](https://cloud.githubusercontent.com/assets/19953651/22386975/7211c0f6-e4a7-11e6-9a0b-0ab5ee5f2fd9.png)

![screen shot 2017-01-27 at 3 41 04 pm](https://cloud.githubusercontent.com/assets/19953651/22386991/7f5ff976-e4a7-11e6-97ac-adc154c30bc7.png)

**Comment by hammondcl** [comment]
@geograil The title of this bug says "Rachel 45", but the text says "Rachel 65". I think it's Rachel 65, so that's what I sent to Greg, but please confirm. 

Also, when I went to check this bug out, there was no event bar over the right handshape entry. Not sure what that might be a result of, but I wanted to mention it in case it is relevant. 
![screen shot 2017-01-30 at 11 21 29 am](https://cloud.githubusercontent.com/assets/15836242/22431125/6dc2ee1a-e6de-11e6-8ab7-580856c25664.png)

**Comment by geograil** [comment]
It is 65. Thanks!

**Comment by gregorydimitriadis** [comment]
I've discovered that this particular gloss suffered from the same problem as was experienced in bugs #104, #105, #108:

(copying the note from those bugs here as well)

"There were some issues in the data saved in the annotation files themselves, so to the best of my ability I have corrected any dropped or missing information, but some of it is lost. For example, for most of the handshape drops, the data is missing and can't be retrieved based solely on the annotation file, so those glosses must be re-created. Once they are re-created, the dropping issue should not reoccur."

What this means particular for this gloss FINISH in this collection is this: the right hand entity was not corresponding correctly to its "parent" gloss1 entity - and so it was behaving strangely in movements and locking/unlocking logic. Between opening the collection and attempting to edit that gloss directly, there's no way to intervene and "error-correct" the incorrect data (since Signstream has every reason to think it is correct.) It must be manually edited in order to resolve the issue.
